.github/workflows/ci.yml

```diff
@@ -58,16 +58,20 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: Test Linux py37
+          - name: Test py37
             os: ubuntu-latest
             pyversion: '3.7'
-          - name: Test Linux py38
+          - name: Test py38
             os: ubuntu-latest
             pyversion: '3.8'
-          - name: Test Linux py39
+          - name: Test py39
             os: ubuntu-latest
             pyversion: '3.9'
-          - name: Test Linux py310
+          - name: Test examples py39
```

Comment:
If we make this a separate job (like the lint-build above), it would get rid of most the `if`'s. I think the `Install llvmpipe` step is also only needed for this build. We could also do this in a later PR to get the merges quicker.

Reply:
Done!